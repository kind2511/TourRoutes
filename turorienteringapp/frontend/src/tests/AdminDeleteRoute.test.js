import React from 'react';
import { render } from '@testing-library/react';
import DeleteRouteButton from '../components/AdminDeleteRoute';

test('DeleteRouteButton renders without crashing', () => {
  render(<DeleteRouteButton routeId={1} onDelete={() => {}} />);
});

//--------------------------------------->

// Snapshot test for the DeleteRouteButton 
test('DeleteRouteButton component snapshot', () => {
  const { asFragment } = render(<DeleteRouteButton routeId={1} onDelete={() => {}} />);
  expect(asFragment()).toMatchSnapshot();
});