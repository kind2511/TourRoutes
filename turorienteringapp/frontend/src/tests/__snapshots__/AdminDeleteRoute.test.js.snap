// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteRouteButton component snapshot 1`] = `
<DocumentFragment>
  <button
    class="admin-routes-delete-button"
  >
    Delete
  </button>
</DocumentFragment>
`;
